pkg/controller/velero/velero.go

```diff
@@ -167,7 +162,7 @@ func (r *ReconcileVelero) provisionVelero(reqLogger logr.Logger, namespace strin
 
 	// Install Deployment
 	foundDeployment := &appsv1.Deployment{}
-	deployment := veleroDeployment(namespace, r.driver.GetPlatformType(), determineVeleroImageRegistry(r.driver.GetPlatformType(), locationConfig["region"]))
+	deployment := veleroDeployment(namespace, r.driver.GetPlatformType(), veleroImageRegistry)
```

Comment:
We can likely just drop this from the internal function completely
```suggestion
	deployment := veleroDeployment(namespace, r.driver.GetPlatformType())
```

As well as on L367.